Without this patch, terminator's build script runs gtk-update-icon-cache which
causes terminator package to claim /usr/share/icons/hicolor/icon-theme.cache as
its own. To avoid that, gtk-update-icon-cache is run later, as part of
xdg_pkg_postinst in the ebuild.

--- a/setup.py	2009-08-12 22:22:53.000000000 -0400
+++ b/setup.py	2009-08-12 22:22:57.000000000 -0400
@@ -25,7 +25,7 @@
 
   def __init__ (self, *args):
     self.without_gettext = False
-    self.without_icon_cache = False
+    self.without_icon_cache = True
     Distribution.__init__(self, *args)
 
 
